scss/_utilities/_clearfix.scss

```diff
@@ -0,0 +1,19 @@
+// ==========================================================================
+// Clearfix
+// ==========================================================================
+
+
+@mixin clearfix() {
+	&:before,
+	&:after {
+		content: "";
+		display: table;
+	}
+	&:after {
+		clear: both;
+	}
+}
+
+.clearfix {
```

Comment:
> Do we use it a whole bunch?

I know I do and many others do. We're also supporting IE8 plus currently

`overflow` is definitely what I use when I can, but it can cause unexpected side effects (scroll bars, positioning issues, cut off child elements). They aren't the worst side effects in the world, but they're there. [#](http://bocoup.com/weblog/new-overflow-default/)

It currently does output `.clearfix` in the output file. This can change.